packages/devtools_app/test/memory/memory_screen_test.dart

```diff
@@ -5,6 +5,8 @@
 import 'package:collection/collection.dart' show IterableExtension;
```

Comment:
Please, move profile related tests to the folder 'profile'.
The memory screen contains a lot of features. We do not want to have all of them in one file.
So, let's organize tests by pane.